apps/frontend-web/app/login/page.tsx

```diff
@@ -42,18 +50,43 @@ export default function Page() {
     mode: 'onChange',
   });
```

Comment:
separate to validation file

Reply:
resolved, using validation.tsx file in login folder